Lake/Config/Glob.lean

```diff
@@ -13,12 +13,12 @@ namespace Lake
 
 /-- A specification of a set of module names. -/
 inductive Glob
-| /-- Selects just the specified module name. -/
-  one : Name → Glob
-| /-- Selects all submodules of the specified module, but not the module itself. -/
-  submodules : Name → Glob
-| /-- Selects the specified module and all submodules. -/
-  andSubmodules : Name → Glob
```

Comment:
Thanks for the PR!

However, please do not restyle the indentation of inductives (here or in `LeanConfig`). There should be no indent before the `|`, (One reason for this style choice is that it makes structure fields, inductive constructors, and tactics line up in the same column, which I find aesthetically pleasing.)